scripts/check-rollup-globals.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Script that goes through each source file that will be included in the
+ * release output and checks if any module imports are not part of the
+ * specified rollup globals file.
+ *
+ * This script is used to validate Bazel rollup globals. We use a genrule to
+ * convert the Starlark rollup globals dict into a JSON file which then can
+ * be passed to this script to ensure that the rollup globals are up-to-date.
+ */
+
+import {green, red, yellow} from 'chalk';
```

Comment:
nit: It looks like the `chalk` coloring isn't working on Circle

Reply:
I'm not sure why that happens. Looks like the gulp messages are not colored either. Might be some weird ANSI rendering logic in the CircleCI UI. I don't think it's worth investigating yet.